project/plugins.sbt

```diff
@@ -2,7 +2,8 @@ resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefac
 resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.15.0")
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.2.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.20")
-addSbtPlugin("org.scoverage"     % "sbt-scoverage"      % "1.9.3")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"        % "3.15.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables"    % "2.2.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.20")
+addSbtPlugin("org.scoverage"     % "sbt-scoverage"         % "1.9.3")
```

Comment:
Probably needs the bump to 2.0.9 and the whole evergreening as part of this (or maybe next?) 🤷‍♂️